api/src/main/scala/Router.scala

```diff
@@ -0,0 +1,126 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.http.scaladsl.model.HttpEntity
```

Comment:
Some of these imports are never used. I would prefer to work minimally, and only add them when needed. Consider removing `HttpEntity`, `spray.json._`, `CorsDirectives`, `JavaConversions`, `Future`, `ExecutionContext`, and `ConfigFactory`.